README.rst

```diff
@@ -112,6 +112,26 @@ For example:
 ``travel()`` instances are nestable, but you'll need to be careful when manually managing to call their ``stop()`` methods in the correct order, even when exceptions occur.
 It's recommended to use the decorator or context manager forms instead, to take advantage of Python features to do this.
 
+Usage with ``shift()`` method
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``start()`` method and entry of the context manager both return a ``Coordinates`` object that corresponds to the time travel.
+This has a ``shift()`` method that takes one argument, ``delta``, which moves the current time.
+``delta`` may be a ``timedelta`` or a number of seconds, which will be added to destination.
+
+For example:
+
+.. code-block:: python
+
+    import datetime as dt
+    import time_machine
+
+    with time_machine.travel(0, tick=False) as traveller:
+        assert time.time() == 0
+
+        traveller.shift(dt.timedelta(seconds=100))
+        assert time.time() == 101
```

Comment:
Sorry, I think this is the last one 😄 

```suggestion
        assert time.time() == 100
```

Reply:
I should really just read it shouldn't I? :P